#! /bin/sh /usr/share/dpatch/dpatch-run
## 06keyid.dpatch by  <az@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: better key-id documentation as requested in #671810

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' duplicity-0.6.18~/bin/duplicity.1 duplicity-0.6.18/bin/duplicity.1
--- duplicity-0.6.18~/bin/duplicity.1	2012-05-07 15:56:27.822124994 +1000
+++ duplicity-0.6.18/bin/duplicity.1	2012-05-07 16:15:15.136826773 +1000
@@ -290,9 +290,13 @@
 Calculate what would be done, but do not perform any backend actions
 
 .TP
-.BI "--encrypt-key " key
+.BI "--encrypt-key " key-id
 When backing up, encrypt to the given public key, instead of using
 symmetric (traditional) encryption.  Can be specified multiple times.
+The key-id can be given in any of the formats supported by GnuPG; see 
+.BR gpg (1),
+section "HOW TO SPECIFY A USER ID" for details.
+
 
 .TP
 .BI "--encrypt-secret-keyring " filename
@@ -304,12 +308,12 @@
 default secret keyring is used which is usually located at .gnupg/secring.gpg
 
 .TP
-.BI "--encrypt-sign-key " key
+.BI "--encrypt-sign-key " key-id
 Convenience parameter. Same as
 .BR --encrypt-key 
-.IR key 
+.IR key-id
 .BR --sign-key 
-.IR "key" .
+.IR "key-id" .
 
 .TP
 .BI "--exclude " shell_pattern
@@ -620,14 +624,15 @@
 sftp client program.
 
 .TP
-.BI "--sign-key " key
+.BI "--sign-key " key-id
 This option can be used when backing up, restoring or verifying. 
 When backing up, all backup files will be signed with keyid
 .IR key .
 When restoring, duplicity will signal an error if any remote file is
-not signed with the given keyid.
-.I key
-should be an 8 character hex string, like AA0E73D2. 
+not signed with the given key-id. The key-id can be givein in any of 
+the formats supported by GnuPG; see 
+.BR gpg (1),
+section "HOW TO SPECIFY A USER ID" for details.
 Should be specified only once because currently only 
 .B one
 signing key is supported. Last entry overrides all other entries.
